c8run/README.md

```diff
@@ -0,0 +1,34 @@
+
+# C8 Run
+
+This is a deployment method very similar to C7 Run.
```

Comment:
I would propose that we add a description that is understandable without knowing C7 Run.

For example

`C8 Run is a packaged distribution of Camunda 8, which allows you to spin up Camunda 8 within seconds. It packages ElasticSearch, and Connectors. The only prerequisite to run it, is to have a JDK 21+ installed locally.`

Reply:
I would propose that we add a quick descirption: what is the prerequisite to run it (JDK), how to get it (fetch from release assets? / provide link to release assets), what does it run (ElasticSearch, Connectors, C8 core), what does it not do (Identity), what is the core use-case (local development). 

Alternative: 
If all of the above is documented in the docs.camunda.io, then just link to that documentation place.

Reply:
That's a good idea.  That README hasn't been updated since we've added the docs.  I'll update the readme to include a brief non-c7run description and link to the documentation.

Reply:
Please review the updated README, as I've made changes.